### ISSUE TITLE
Can't reproduce results on Vimeo-90K

### ISSUE THREAD

**Issue by tarun005** [JunHeum/BMBC#6]
Hi,

I downloaded your pretrained model, and calculated PSNR on the test split of Vimeo-90K. I obtained a PSNR of only 30.544 compared to 35.01 in your paper. 

Can you help me what could have gone wrong? What are the specific pre-processing steps you followed (normalization, crop size) during testing? Is the available model the one used to report the final results in the paper?

If possible, can you share the exact script you used to benchmark your results in the paper?

**Comment by tarun005** [comment]
Using PyTorch 1.3.1 solved the issue! It turns out few grid sampling techniques used in the code are highly sensitive to the PyTorch version used.
